bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/XMLAttributes.java

```diff
@@ -60,6 +60,9 @@ public boolean equals(Object obj) {
         if (obj == null || !(obj instanceof XMLAttributes)) {
             return false;
         }
+        ArrayValue ara = new ArrayValue(BTypes.typeMap, 5);
+        ara.add(0,new MapValue<>());
+        MapValue val = (MapValue) ara.getRefValue(0);
```

Comment:
Do we need this code snippet?

Reply:
Oops, yes that's by mistake. Removed now.